WMF Framework/WidgetController.swift

```diff
@@ -63,11 +63,10 @@ public final class WidgetController: NSObject {
     }
     
     private func fetchCachedWidgetContentGroup(with kind: WMFContentGroupKind, isAnyLanguageAllowed: Bool, in dataStore: MWKDataStore, completion:  @escaping (WMFContentGroup?) -> Void) {
+        assert(Thread.isMainThread, "Cached widget content group must be fetched from the main queue")
         let moc = dataStore.viewContext
```

Comment:
The implementation of `dataStore.viewContext` assumes it's always called from the main thread, so just ensure this whole function is called on the main thread and remove the `moc.perform`